### DISCUSSION TITLE
add giscus to mkdocs-material official document website

### DISCUSSION THREAD

**Discussion by makerjackie** [squidfunk/mkdocs-material#3807] [Ideas]
giscus is great!!! why not add giscus to https://squidfunk.github.io/mkdocs-material website?

**Comment by squidfunk** [comment]
We have added it in the [Blog](https://squidfunk.github.io/mkdocs-material/blog/2021/the-past-present-and-future/#__comments) section.

**Reply by makerjackie** [reply]
ok, great, I wonder if there is a better hints to info user ` you need to login first to comment`  from the begining,  instead of scroll to the end of page to find `sign with github` button
![image](https://user-images.githubusercontent.com/18050469/162695746-1ea759a9-e03c-4b6b-aff5-6453f214c4c4.png)

**Reply by squidfunk** [reply]
This is rendered by Giscus, so better ask the maintainers of Giscus 😊
